Task_1/setup.py

```diff
@@ -24,15 +24,14 @@
     url='https://github.com/FETS-AI/Challenge',
     packages=[
         'fets_challenge',
-        'openfl-workspace',
     ],
     include_package_data=True,
     install_requires=[
-        'openfl @ git+https://github.com/intel/openfl.git@f4b28d710e2be31cdfa7487fdb4e8cb3a1387a5f',
-        'GANDLF @ git+https://github.com/CBICA/GaNDLF.git@e4d0d4bfdf4076130817001a98dfb90189956278',
+        'openfl @ git+https://github.com/securefederatedai/openfl.git@v1.7.1',
```

Comment:
Instead of cloning the repo, I suggest directly consuming the openfl package from PyPI (as in `openfl==1.7.1`).